packages/accounts-controller/package.json

```diff
@@ -62,18 +62,22 @@
   "devDependencies": {
     "@metamask/auto-changelog": "^3.4.4",
     "@metamask/keyring-controller": "^19.0.0",
+    "@metamask/providers": "^18.1.1",
     "@metamask/snaps-controllers": "^9.10.0",
     "@types/jest": "^27.4.1",
     "@types/readable-stream": "^2.3.0",
     "jest": "^27.5.1",
     "ts-jest": "^27.1.4",
     "typedoc": "^0.24.8",
     "typedoc-plugin-missing-exports": "^2.0.0",
-    "typescript": "~5.2.2"
+    "typescript": "~5.2.2",
+    "webextension-polyfill": "^0.12.0"
   },
   "peerDependencies": {
     "@metamask/keyring-controller": "^19.0.0",
-    "@metamask/snaps-controllers": "^9.7.0"
+    "@metamask/providers": "^18.1.0",
+    "@metamask/snaps-controllers": "^9.7.0",
+    "webextension-polyfill": "^0.10.0 || ^0.11.0 || ^0.12.0"
```

Comment:
I have copied the ranges for these peer dependencies from the packages that require them instead of using new ranges.